test/index.spec.js

```diff
@@ -173,6 +173,23 @@ describe('CID', () => {
       expect(cid.prefix.toString('hex')).to.equal('01711220')
     })
 
+    it('.prefix identity multihash', () => {
+      const mh = multihash.encode(Buffer.from('abc'), 'identity')
+      const cid0 = new CID(0, 'dag-pb', mh)
+
+      expect(cid0).to.have.property('codec', 'dag-pb')
+      expect(cid0).to.have.property('version', 0)
+      expect(cid0).to.have.property('multihash').that.eql(mh)
+      expect(cid0.toBaseEncodedString()).to.eql('161g3c')
```

Comment:
FYI (this is not blocking a merge): you can also use `toString()` instead of `toBaseEncodedString()`.